scipy/stats/_qmc.py

```diff
@@ -1594,3 +1591,35 @@ def reset(self) -> MultinomialQMC:
         super().reset()
         self.engine.reset()
         return self
+
+
+def _validate_workers(workers: IntNumber = 1) -> IntNumber:
+    """
+    Perfom check on worker param and return the new optionally modified worker
+    value to be compliant with the other functions
```

Comment:
IMO a bit too verbose
```suggestion
    """Validate `workers` based on platform and value.
```